lib/stytch/b2b_magic_links.rb

```diff
@@ -14,6 +20,80 @@ def initialize(connection)
       @discovery = StytchB2B::MagicLinks::Discovery.new(@connection)
     end
 
+    # Authenticate a Member with a Magic Link. This endpoint requires a Magic Link token that is not expired or previously used. Provide a value for `session_duration_minutes` to receive a Session. If the Member’s status is `pending` or `invited`, they will be updated to `active`.
+    #
+    # == Parameters:
+    # magic_links_token::
+    #   The Email Magic Link token to authenticate.
+    #   The type of this field is `String`.
+    # pkce_code_verifier::
+    #   A base64url encoded one time secret used to validate that the request starts and ends on the same device.
+    #   The type of this field is nilable `String`.
+    # session_token::
+    #   Reuse an existing session instead of creating a new one. If you provide a `session_token`, Stytch will update the session.
+    #       If the `session_token` and `magic_links_token` belong to different Members, the `session_token` will be ignored. This endpoint will error if
+    #       both `session_token` and `session_jwt` are provided.
+    #   The type of this field is nilable `String`.
+    # session_jwt::
+    #   Reuse an existing session instead of creating a new one. If you provide a `session_jwt`, Stytch will update the session. If the `session_jwt`
+    #       and `magic_links_token` belong to different Members, the `session_jwt` will be ignored. This endpoint will error if both `session_token` and `session_jwt`
+    #       are provided.
+    #   The type of this field is nilable `String`.
+    # session_duration_minutes::
+    #   Set the session lifetime to be this many minutes from now. This will start a new session if one doesn't already exist,
+    #   returning both an opaque `session_token` and `session_jwt` for this session. Remember that the `session_jwt` will have a fixed lifetime of
+    #   five minutes regardless of the underlying session duration, and will need to be refreshed over time.
+    #
+    #   This value must be a minimum of 5 and a maximum of 527040 minutes (366 days).
+    #
+    #   If a `session_token` or `session_jwt` is provided then a successful authentication will continue to extend the session this many minutes.
+    #
+    #   If the `session_duration_minutes` parameter is not specified, a Stytch session will be created with a 60 minute duration. If you don't want
+    #   to use the Stytch session product, you can ignore the session fields in the response.
+    #   The type of this field is nilable `Fixnum`.
```

Comment:
Should we be referencing [Fixnum](https://stackoverflow.com/questions/21372649/what-is-the-difference-between-integer-and-fixnum) still?